runtime/doc/cmdline.txt

```diff
@@ -56,7 +56,7 @@ history tables:
 These are completely separate.  Each history can only be accessed when
 entering the same type of line.
 Use the 'history' option to set the number of lines that are remembered
-(default: 20).
+(default: 50).
```

Comment:
This is consistent with the default mentioned in `runtime/doc/options.txt`. However, Neovim's default still seems to be the previous value (`20`). Vim 7.4.537 already has the default of `50`.